.drone.yml

```diff
@@ -0,0 +1,61 @@
+---
+kind: pipeline
+name: amd64
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: build
+  image: autonomy/build-container:latest
+  pull: always
+  environment:
+    PLATFORM: linux/amd64
+  commands:
+    - docker context create ci-amd64 --docker "host=tcp://docker-amd64.ci.svc:2375"
```

Comment:
just found out that we don't actually need this and its more intended for human usage I think. Here is what I'm doing in the Talos build refactor instead:

```
docker buildx create --name ci --driver docker-container --use tcp://docker-amd64.ci.svc:2375
docker buildx create --name ci --driver docker-container --append tcp://docker-arm64.ci.svc:2375
``